astro-docs/src/content/docs/technologies/dotnet/Guides/migrate-from-nx-dotnet-core.mdoc

```diff
@@ -0,0 +1,448 @@
+
+```json
+{
+  "name": "my-api",
+  "targets": {
+    "build": {
+      "command": "dotnet build",
+      "options": {
+        "cwd": "{projectRoot}",
+        "args": ["--configuration", "Release", "--no-dependencies"]
+      }
+    },
+    "test": {
+      "command": "dotnet test MyApi.Tests.csproj",
+      "options": {
+        "cwd": "{projectRoot}"
+      }
+    },
+    "serve": {
+      "command": "dotnet watch run",
+      "options": {
+        "cwd": "{projectRoot}"
+      }
+    }
+  }
+}
+```
+
+{% callout type="note" title="Automatic Target Inference" %}
+In most cases, you can delete these `project.json` files entirely. The @nx/dotnet plugin will automatically infer the correct targets based on your `.csproj` files. Only keep custom `project.json` configurations if you need non-standard options.
+{% /callout %}
+
+### Migrating from `inferProjects: false`
+
+If you previously set `inferProjects: false` in your @nx-dotnet/core config, this option no longer exists in @nx/dotnet. The new plugin **always** infers projects automatically.
+
+**Why this changed:**
+
+- Project inference is a core feature of how Nx plugins work
+- Manual project management doesn't align with Nx's automatic detection model
+- Maintaining explicit project configurations is error-prone
+
+**Migration path:**
+
+1. **If you disabled inference to exclude certain projects:**
+
+   - Use the `exclude` property in the plugin configuration:
+     ```json
+     {
+       "plugins": [
+         {
+           "plugin": "@nx/dotnet",
+           "exclude": ["legacy-projects/**", "temp-projects/**"]
+         }
+       ]
+     }
+     ```
+
+2. **If you disabled inference to use explicit project.json files:**
+
+   - The new plugin will still detect your projects via `.csproj` files
+   - You can keep `project.json` files to customize targets, but they're optional
+   - Consider removing them to use the automatically inferred targets
+
+3. **If you need to completely hide projects from Nx:**
+   - Use the `exclude` property in the plugin configuration
+   - Or move projects outside the workspace
+
+## Configuration Changes
+
+### Module Boundaries → Nx Conformance
+
+If you used `moduleBoundaries` in your @nx-dotnet/core config, you may want to migrate to [Nx Conformance rules](https://nx.dev/docs/enterprise/powerpack/conformance)
+
+The script used to evaluate module boundaries is missing several features from the eslint and conformance implementations, and is not present in the new plugin. If you wish to continue using it, you'll need to inline the implementation in your repository and update the path in `Directory.build.targets`. Otherwise, you should remove the target from `Directory.build.targets`.
+
+### NuGet Package Management
+
+Replace the `sync` generator with MSBuild's [Central Package Management](https://learn.microsoft.com/nuget/consume-packages/central-package-management).
+
+Create `Directory.Packages.props` at workspace root:
+
+```xml
```

Comment:
add example file name (I think it's like project-name.csproj?) 
```suggestion
```xml
// project.csproj
```